vite-project-kabinet/src/Content/Card.jsx

```diff
@@ -0,0 +1,57 @@
+import "./card.style.css"
+
+export const Card = ({genre, title, author, content, image}) => {
+
+    const displayImage = () => {
```

Comment:
Tohle by měla být komponenta a ne funkce

```suggestion
  const Image = (props) => {
  ....
  
```